app/assets/stylesheets/components/_modal.scss

```diff
@@ -48,23 +48,33 @@
   .modal-content {
     position: relative;
     width: 100%;
-    max-height: 85vh;
-    padding: $box-space;
+    max-height: 95vh;
```

Comment:
We can't use vh in mobile devices. It generates a lot of issues in iOS devices. Try to use percentage instead of vh. Let me know if you have any question